docs/getting-started/installation-and-upgrade/installation-references/feature-flags.md

```diff
@@ -49,3 +50,4 @@ The following table shows the availability and default values for some feature f
 | `token-hashing` | `false` for new installs, `true` for upgrades | GA | v2.6.0 | |
 | `uiextension` | `true` | GA | v2.9.0 | |
 | `ui-sql-cache` | `false` | Highly experimental | v2.9.0 | |
+| `clean-stale-secrets` | `true` | FA | v2.9.6 | |
```

Comment:
Is FA a typo (and should be GA) or is a different term we're introducing?

Reply:
@btat Thank you! fixed